### DISCUSSION TITLE
Deselect packages in group:allNonMajor PR

### DISCUSSION THREAD

**Discussion by runeh** [renovatebot/renovate#22144] [Suggest an Idea]
We would like to remove one or more packages from an open PR that updates packages in the `group:allNonMajor`

Our problem is that we know certain packages with minor and patch releases that have issues, and can't be bumped in our monorepo at the moment.

Would be nice if we could have checkboxes next to the packages to be updated in the PR, and be able to uncheck a box to have it be removed from the PR. Perhaps as a bonus, create a standalone PR for each unchecked package.

**Comment by rarkins** [comment]
This is a good example of why not to group too aggressively - it’s always a trade off/risk.

Unfortunately there’s no easy way I can think of to implement what you ask for but I’ll keep this request open in case one day that changes!

You could add package rules with allowedVersions set to exclude the version you want to skip, but it’s not too convenient.

**Comment by rarkins** [comment]
Thinking this through from an internals point of view:

- If there were an `allowedVersions` package rule for each affected package then those "bad" updates would be skipped anyway and not included in the "all" PR

- If a package rule for those packages was added and set `groupName` to `null` then they'd be split out and in their own PRs - permanently
